### DISCUSSION TITLE
when support .net core 3.1

### DISCUSSION THREAD

**Discussion by easyboot** [cefsharp/CefSharp#4093] [General]
![image](https://user-images.githubusercontent.com/26960023/167232741-e6d18135-4a8f-4983-9434-87d9a109116d.png)

I Use Cefsharp,but IDE warning NU1701.

**Comment by amaitland** [comment]
Firstly please use https://github.com/cefsharp/CefSharp/discussions to ask questions.

Secondly support for .Net Core 3.1 was added over a year ago in #2796

You need to use a different `Nuget` package, see https://github.com/cefsharp/CefSharp/wiki/Quick-Start#installation-ms-net-core-31net-50net-60 for details.
